<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Knjige</class>
 <widget class="QDialog" name="Knjige">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Knjige</string>
  </property>
  <widget class="QLabel" name="labelSifra">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sifra</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUDK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>UDK</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelISBN">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNaziv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Naziv</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSifra">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUDK">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditISBN">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNaziv">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpisi">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Upisi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonObrisi">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Obrisi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIzmeni">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Izmeni</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIzadji">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Izadji</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>580</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
